source/brailleDisplayDrivers/papenmeier.py

```diff
@@ -162,7 +139,7 @@ def connectBluetooth(self):
 							log.debugWarning("connectBluetooth failed")
 
 	def connectUSB(self, devlist: List[bytes]):
-		"""try to connect to usb device,is triggered when BRXCOM is not installed and bluetooth
+		"""try to connect to usb device,is triggered when bluetooth

```

Comment:
Can we fix the missing space and wording here?

```suggestion
		"""Try to connect to usb device, this is triggered when bluetooth
```

Reply:
I committed the suggesttion. Or didn't I?
Why is this marked as "outdated"?